src/core/src/op/inverse.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/inverse.hpp"
+
+#include <cstring>
+
+#include "inverse_shape_inference.hpp"
+#include "itt.hpp"
+#include "openvino/core/attribute_visitor.hpp"
+#include "openvino/op/util/op_types.hpp"
+#include "openvino/reference/inverse.hpp"
+
+namespace ov {
+
+// ------------------------------ v14 ------------------------------
```

Comment:
Instead you can put all in
```suggestion
namespace op{
namespace v14{
```

Reply:
Right, moved!

Reply:
There is no change, or I missed it?